### ISSUE TITLE
can't resolve CSS file with custom workspace

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#85]
## Type of Change
- [ ] New Feature Request
- [ ] Documentation / Website
- [ ] Improvement / Suggestion
- [x] Bug
- [ ] Other (please clarify below)

## Summary
Trying to create a project with a custom page template and shared CSS file and getting a build error.  (related to #80 )

```shell
$ yarn build
yarn run v1.12.3
$ yarn clean && node ./packages/cli/index.js build
$ rimraf ./**/.greenwood/** && rimraf ./**/public/** && rimraf ./coverage
-------------------------------------------------------
Welcome to Greenwood ♻️
-------------------------------------------------------
Reading project config
Initializing project workspace contexts
Generating graph of workspace files...
Scaffolding out project files...
Generate pages from templates...
Writing imports for md...
Writing Lit routes...
setup index page and html
Scaffolding complete.
Building project for production.
Building SPA from compilation...
./.greenwood/about/about.js
Module not found: Error: Can't resolve '../theme.css' in '/Users/owenbuckley/Workspace/project-evergreen/repos/greenwood/.greenwood/about'
```

## Details
Workspace
```shell
www/
   pages/
     index.md
     about.md
    templates/ 
      page-template.js
    theme.css
```

Config
```javascript
const path = require('path');

module.exports = {
  workspace: path.join(__dirname, 'www'),
  title: 'Greenwood',
  meta: [
    { property: 'og:site', content: 'greenwood' },
    { name: 'twitter:site', content: '@PrjEvergreen' }
  ]
};
```

> Note: I wasn't able to reproduce in the _build.default.workspace-template-page-style_ even after adding a _pages/_ directory with an _index.md_ and _about.md_ so I think it could be related to using a custom workspace?

**Comment by thescientist13** [comment]
I think we've covered this through the docs that everything needs to be in a directory.  Not sure what the effort is to support this or if it's worth it?

**Comment by thescientist13** [comment]
I think this is resolved now with #355 , maybe just need to make sure there is a test for this

**Comment by thescientist13** [comment]
not sure this is an issue anymore in v0.10.0.
